### PR TITLE
Course

### PR DESCRIPTION
**Pull request by CollinBrady1993**
my additions

### CODE REVIEW

File: model/simple-wireless-net-device.cc [hunk 1]
```diff
@@ -315,12 +320,12 @@ SimpleWirelessNetDevice::Receive (Ptr<Packet> packet, double rxPower, uint16_t p
 
   m_phyRxEndTrace (packet, rxPower, from);
 
-  double noisePower = -100; // placeholder -- to be set by attribute
+  //double noisePower = -100; // placeholder -- to be set by attribute
```
Comment (tomhenderson) [inline model/simple-wireless-net-device.cc]: Delete statement, do not comment out

File: model/simple-wireless-net-device.cc [hunk 2]
```diff
@@ -918,6 +923,13 @@ SimpleWirelessNetDevice::SetDataRate (DataRate bps)
   m_bps = bps;
 }
 
+void
+SimpleWirelessNetDevice::SetNoisePower (double noisePower)
+{
+  NS_LOG_FUNCTION_NOARGS ();
```
Comment (tomhenderson) [inline model/simple-wireless-net-device.cc]: should be 'NS_LOG_FUNCTION (this << noisePower);'

File: model/simple-wireless-net-device.h [hunk 3]
```diff
@@ -176,6 +176,11 @@ class SimpleWirelessNetDevice : public NetDevice
  * @see Queue
  * @see DropTailQueue
  * @param queue Ptr to the new queue.
+ */
+ void SetNoisePower (double noisePower);
+
+  /**
+ * set noise power
```
Comment (tomhenderson) [inline model/simple-wireless-net-device.h]: SetNoisePower () should be below the doxygen comments, not above.

**Review by tomhenderson — CHANGES_REQUESTED** [review]
some minor changes needed

**Comment by CollinBrady1993** [comment]
made corrections, hopefully I dont need to submit a new pull request

**Pull request merged**